modules/react-native-mapbox-maps/package.json

```diff
@@ -0,0 +1,15 @@
+{
+	"name": "@modules/mapbox-maps",
+	"version": "1.0.0",
+	"description": "Add interactive high customized maps to your mobile app to make it easier for users to find your locations.",
+	"private": true,
+	"main": "index.js",
+	"x-dependencies": {
+		"@rnmapbox/maps": "rnmapbox/maps#main",
+		"@turf/helpers": "^6.5.0",
+		"react-native-switch-toggle": "^2.2.1",
+		"react-native-vector-icons": "^8.1.0"
```

Comment:
`react-native-vector-icons` is already included in the scaffold, no need to add it here

Reply:
Done.